_posts/2015-09-04-channel.html

```diff
@@ -435,6 +435,272 @@
+			<tr>
+				<td>device_name</td>
+				<td>string</td>
+				<td>Name of the device password</td>
+			</tr>
+			<tr>
+				<td>user_name</td>
+				<td>string</td>
+				<td>Username of the device password</td>
+			</tr>
+			<tr>
+				<td>created_at</td>
+				<td>timestamp</td>
+				<td>Date of creating</td>
+			</tr>
+			<tr>
+				<td>valid_from</td>
+				<td>timestamp</td>
+				<td>From that time the user can login</td>
+			</tr>
+			<tr>
+				<td>valid_to</td>
+				<td>timestamp</td>
+				<td>Until that time the user can login</td>
+			</tr>
+		</table>
+
+		<h6>Positive response example</h6>
+		<pre><code class="json">{
+  "device_users": [
+    {
+      "device_name": "Sample Device 1",
+      "user_name": "xnwj3mseuka",
+      "created_at": "1497441340",
+      "valid_from": "1497443340",
+      "valid_to": null
+    },
+    {
+      "device_name": "Sample Device 2",
+      "user_name": "ymvbk9bu7f8",
+      "created_at": "1497341340",
+      "valid_from": "1497341880",
+      "valid_to": "1497341880"
+    },
+    {
+      "device_name": "Sample Device 3",
+      "user_name": "rdhj8sd8kar",
+      "created_at": "1491341880",
+      "valid_from": "1492341880",
+      "valid_to": null
+    }
+  ],
+  "paging": []
+}</code></pre>
+
+		<p>The paging information can be found under the "paging" element. Example:</p>
+
+		<pre><code class="json">"paging": {
+	"previous": ".../users/self/deviceusers.json?pagesize=PAGE_SIZE&page=PREVIOUS_PAGE",
+	"next": ".../users/self/deviceusers.json?pagesize=PAGE_SIZE&page=NEXT_PAGE"
+}</code>
+</pre>
+
+		<h6>Error codes</h6>
+
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Unauthorizedt</td>
+				<td>The provided Access Token is missing, revoked, expired or malformed</td>
+			</tr>
+		</table>
+
+		<h3>Create device password</h3>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">POST</td>
+				<td><a href="https://api.ustream.tv/users/self/deviceusers.json">https://api.ustream.tv/users/self/deviceusers.FORMAT</a> (format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Parameters</h6>
+
+		<p>The body of the call should contain the device name and valid from timestamp.</p>
+
+		<table class="table responsive">
+			<tr>
+				<th>parameter</th>
+				<th>type</th>
+				<th>importance</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td><strong>device_name</strong></td>
+				<td>string</td>
+				<td>REQUIRED</td>
+				<td>Name of the device</td>
+			</tr>
+			<tr>
+				<td><strong>valid_from</strong></td>
+				<td>timestamp</td>
+				<td>REQUIRED</td>
+				<td>From that time the user can login</td>
+			</tr>
+			<tr>
+				<td><strong>valid_to</strong></td>
+				<td>timestamp</td>
+				<td>OPTIONAL</td>
+				<td>Until that time the user can login</td>
+			</tr>
+		</table>
+
+
+		<h6>Response</h6>
```

Comment:
A "Response" alatt általában ez szokott állni a doksiban: "On success a response with HTTP status 200 OK is returned."